rotkehlchen/tests/unit/decoders/test_thegraph.py

```diff
@@ -302,7 +307,7 @@ def test_thegraph_undelegate(ethereum_inquirer):
             asset=A_GRT,
             amount=ZERO,
             location_label=ADDY_USER,
-            notes=f'Undelegate {undelegate_amount} GRT from indexer {indexer_address}. Lock expires in {lock_time} seconds',  # noqa: E501
+            notes=f'Undelegate {undelegate_amount} GRT from indexer {indexer_address}. Lock expires at epoch {lock_time}',  # noqa: E501
```

Comment:
from the contract, epoch is not in seconds so the notes is wrong.
https://github.com/graphprotocol/contracts/blob/b91017dc1e3492d768d0df377a27f4b15606950e/packages/contracts/contracts/staking/StakingExtension.sol#L459

the contract is https://arbiscan.io/address/0x5A843145c43d328B9bB7a4401d94918f131bB281#readProxyContract